ci/libs/utils.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+. ci/libs/variables.sh
+
+cleanup(){
+	rm -rf $TMP_DIR/vol*
+	rm -rf $MNT_DIR/logs
+    docker stop $(docker ps -aq)
```

Comment:
indentation issuses also if you could provide just the ids of docker here it will be good

Reply:
you can use xargs to directly use them plus this would stop all docker containers do we really want that?

Reply:
I'll fix the indentation issue. Regarding the other comment, 
Since we are dealing with starting and stopping of multiple containers with multiple IPs and all, I think we should avoid having any running containers for running integration tests. Anyway this is going to take place in travis